app/src/main/java/com/cornellappdev/android/eateryblue/ui/components/home/EateryMenuWidget.kt

```diff
@@ -0,0 +1,192 @@
+package com.cornellappdev.android.eateryblue.ui.components.home
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material.Divider
+import androidx.compose.material.Icon
+import androidx.compose.material.IconButton
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.rememberCoroutineScope
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.res.vectorResource
+import androidx.compose.ui.unit.dp
+import com.cornellappdev.android.eateryblue.R
+import com.cornellappdev.android.eateryblue.data.models.Event
+import com.cornellappdev.android.eateryblue.ui.components.general.SearchBar
+import com.cornellappdev.android.eateryblue.ui.theme.EateryBlueTypography
+import com.cornellappdev.android.eateryblue.ui.theme.GrayFive
+import com.cornellappdev.android.eateryblue.ui.theme.GrayZero
+import java.time.format.DateTimeFormatter
+
+/**
+ * Displays the menu of a particular event based on categories like Grill or Soup
+ * At the top of the widget, the meal description (e.g. Lunch or Dinner) and duration
+ * of opening is displayed.
+ */
+@Composable
+fun EateryMenuWidget(
+    event: Event,
+    hoursOnClick: () -> Unit
+) {
+    rememberCoroutineScope()
+
+    var filterText by remember { mutableStateOf("") }
+
+    Row(
+        modifier = Modifier.padding(top = 16.dp, bottom = 8.dp, start = 16.dp, end = 10.dp),
+        verticalAlignment = Alignment.CenterVertically
+    ) {
+        Column(
+            modifier = Modifier.weight(1f)
+        ) {
+            Text(
+                text = event.description ?: "Full Menu",
+                style = EateryBlueTypography.h4,
+            )
+            if (event.startTime != null && event.endTime != null) {
+                Text(
+                    text = "${event.startTime.format(DateTimeFormatter.ofPattern("K:mm a"))} - ${
+                        event.endTime.format(
+                            DateTimeFormatter.ofPattern("K:mm a")
+                        )
+                    }", style = EateryBlueTypography.subtitle2, color = GrayFive
+                )
```

Comment:
Bug spotted!

![image](https://github.com/cuappdev/eatery-blue-android/assets/47724806/871a03a8-8491-4483-b840-61fbd5d16b61)

According to Chat-GPT:

```
"
- "K" represents hour-of-am-pm (0-11). The hour is represented in a range from 0 to 11, where 0 represents midnight and 11 represents 11 AM.
- "h" represents clock-hour-of-am-pm (1-12). The hour is represented in a range from 1 to 12, where 1 represents 1 AM and 12 represents noon (12 PM)."
```

Change this over to `h` instead of `K` and it should fix! Feel free to use Anabel's as a means of testing this.